new-website/utils/models/get_models.py

```diff
@@ -0,0 +1,143 @@
+"""
+Python script to scrape the all the models information from the official deepchem documentation
+"""
+
+import os
+import json
+import dataclasses
+from json import JSONEncoder
+from bs4 import BeautifulSoup
+import requests
+
+DOC_URL = 'https://deepchem.readthedocs.io/en/latest/api_reference/models.html#model-cheatsheet'
+REDIRECT_URL = 'https://deepchem.readthedocs.io/en/latest/api_reference/models.html#'
+TABLES = {
+    'id81': 'General',
```

Comment:
Discussed offline; better to point to CSV files such as https://github.com/deepchem/deepchem/blob/master/docs/source/api_reference/general_purpose_models.csv

Reply:
Changed the source of data from the documentation to the stored CSV files in the Deepchem repo.